week2/to-do-list/app.js

```diff
@@ -8,4 +8,33 @@ const todayTodoButton = document.querySelector('.today-input-button')
 const tomorrowTodoInput = document.querySelector('.tomorrow-to-do-input')
 const tomorrowTodoButton = document.querySelector('.tomorrow-input-button')
 const todayTodoList = document.querySelector('.today-todo-list')
-const tomorrowTodoList = document.querySelector('.tomorrow-todo-list')
\ No newline at end of file
+const tomorrowTodoList = document.querySelector('.tomorrow-todo-list')
+
+const addTodayTodo = () => {
+    if(todayTodoInput.value == ""){
+        alert("내용을 입력하세요!")
+    }else{
+        const currenLi = document.createElement("li");
+        currenLi.classList.add("todo-item")
+        const creatDeleteButton = document.createElement("span");
+    
+    
+        const currentInputData = todayTodoInput.value;
+        const plusToDoList = document.createElement("span");
+        plusToDoList.innerHTML= `${currentInputData}`
+        creatDeleteButton.className += " material-icons today-delete-button";
+        creatDeleteButton.innerHTML = 'delete'
+    
+        currenLi.appendChild(plusToDoList)
+        currenLi.appendChild(creatDeleteButton)
```

Comment:
```suggestion
currentLi.innerHTML = ` <span>${currentInputData}</span><span class="material-icons delete-button"> delete </span>
`
```

요러케 html 만들고 appendChild 할 수 이따앙ㅎㅎㅎ

Reply:
오오  ` 을 잘 사용하니까  이게 더 간결해졌다!